### PR TITLE
Fill in placeholder values

### PR DESCRIPTION
**Pull request by balghane**
Filled in placeholder values for all the subsystems.

Did not fill in any inversions.

Some placeholders are still there also.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java [hunk 1]
```diff
@@ -9,5 +9,5 @@ public class ConveyorConstants {
   public static final double FRONT_GEAR_RATIO = Constants.PLACEHOLDER_DOUBLE,
       BACK_GEAR_RATIO = Constants.PLACEHOLDER_DOUBLE;
 
-  public static final double CONVEYOR_MOTOR_ROLLER_DIAMETER_IN = Constants.PLACEHOLDER_DOUBLE;
+  public static final double CONVEYOR_MOTOR_ROLLER_DIAMETER_IN = 1.1;
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java:12]: This should be double checked according to the doc; either wait to merge or add a comment

File: src/main/java/frc/robot/subsystems/drive/DriveConstants.java [hunk 2]
```diff
@@ -45,27 +41,19 @@ public class DriveConstants {
...
+  public static final double TRACK_WIDTH_METERS = Units.inchesToMeters(21.5);
 
   /**
    * Driving Parameters - Note that these are not the maximum capable speeds of the robot, rather
-   * the allowed maximum speeds
+   * the allowed maximum speeds.
+   * Max speed is 85% free speed
    */
-  public static final double MAX_SPEED_METERS_PER_SECOND = Constants.PLACEHOLDER_DOUBLE,
+  public static final double MAX_SPEED_METERS_PER_SECOND = 0.85 * Units.feetToMeters(20.25),
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/drive/DriveConstants.java]: I'd separate out the 0.85 into a separate variable but maybe that isn't necessary

File: src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java [hunk 3]
```diff
@@ -1,18 +1,16 @@
 package frc.robot.subsystems.elevator;
 
-import frc.robot.Constants;
-
 public final class ElevatorConstants {
   public static final int MAX_INIT_RETRY_ATTEMPTS = 5;
 
-  public static final double ELEVATOR_DRUM_DIAMETER_IN = Constants.PLACEHOLDER_DOUBLE;
+  public static final double ELEVATOR_DRUM_DIAMETER_IN = 1.0;
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java:6]: Is this the same as the winch?

File: src/main/java/frc/robot/subsystems/intake/Intake.java
Comment (tails618) [inline src/main/java/frc/robot/subsystems/intake/Intake.java:1]: Confirm this?
image

File: src/main/java/frc/robot/subsystems/intake/IntakeCal.java [hunk 4]
```diff
@@ -11,33 +11,36 @@ public class IntakeCal {
...
-      INTAKE_STOWED_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE,
-      INTAKE_SAFE_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE;
+  public static final double INTAKE_DEPLOYED_POSITION_DEGREES = 270.0,
+      INTAKE_STOWED_POSITION_DEGREES = 155.0,
+      INTAKE_SAFE_POSITION_DEGREES = 200.0; // exact safe position is 186.0
 
-  public static final double INTAKE_MARGIN_DEGREES = Constants.PLACEHOLDER_DOUBLE,
-      CONVEYOR_ZONE_THRESHOLD_DEGREES = Constants.PLACEHOLDER_DOUBLE;
+  public static final double INTAKE_MARGIN_DEGREES = 1.0,
+      CONVEYOR_ZONE_THRESHOLD_DEGREES = 190.0;
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/intake/IntakeCal.java]: How is this different from `INTAKE_SAFE_POSITION_DEGREES`?

File: src/main/java/frc/robot/subsystems/shooter/ShooterCal.java [hunk 5]
```diff
@@ -20,38 +20,43 @@ public class ShooterCal {
...
   /** How close to goal position to be considered "there" */
-  public static final double PIVOT_ANGLE_MARGIN_DEG = Constants.PLACEHOLDER_DOUBLE;
+  public static final double PIVOT_ANGLE_MARGIN_DEG = 0.5;
 
   /** Target shooter motor speeds (at the motor) */
-  public static final double SHOOTER_MOTOR_SPEED_RPM = Constants.PLACEHOLDER_DOUBLE;
+  public static final double SHOOTER_MOTOR_SPEED_RPM = 4000.0;
+
+  /** What the abs encoder reads (in degrees) when the shooter is pointed down */
+  public static final double PIVOT_ANGLE_OFFSET_DEGREES = Constants.PLACEHOLDER_DOUBLE;
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/shooter/ShooterCal.java]: How can we determine this? Can the shooter be pointed down?

**Review by tails618 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java:12]: This one I won't think will matter a whole lot tbh. Like if the deviation is 1/8" or something it doesn't matter a whole lot

File: src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java (hunk 3 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java:6]: Ya

File: src/main/java/frc/robot/subsystems/shooter/ShooterCal.java (hunk 5 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/shooter/ShooterCal.java]: Point it at 90 and read the encoder value

File: src/main/java/frc/robot/subsystems/intake/IntakeCal.java (hunk 4 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/intake/IntakeCal.java]: Added comment

File: src/main/java/frc/robot/subsystems/intake/Intake.java
Comment (balghane) [reply src/main/java/frc/robot/subsystems/intake/Intake.java:1]: Added todo to double check

File: src/main/java/frc/robot/subsystems/drive/DriveConstants.java (hunk 2 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/drive/DriveConstants.java]: Nah fair call, changed

File: src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java (hunk 1 above)
Comment (tails618) [reply src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java:12]: Good point

File: src/main/java/frc/robot/subsystems/shooter/ShooterCal.java (hunk 5 above)
Comment (tails618) [reply src/main/java/frc/robot/subsystems/shooter/ShooterCal.java]: Comment says "when shooter is pointed down" - 90deg is horizontal

**Review by tails618 — APPROVED** [review]
Clarify that comment; everything else is good

**Pull request merged**